release.py

```diff
@@ -62,8 +62,10 @@ def create_data_schema_documentation():
 
 def make_release_specific_changes(release_version):
     execute_steps([
-        "python ./web_frontend/manage.py makemessages -l de_CH -l en -l en_UK -l en_US",
-        "python ./osmaxx_conversion_service/manage.py makemessages -l de_CH -l en -l en_UK -l en_US",
+        "python ./web_frontend/manage.py makemessages -l en",
+        "python ./osmaxx_conversion_service/manage.py makemessages -l en",
+        "python ./web_frontend/manage.py compilemessages -l en",
+        "python ./osmaxx_conversion_service/manage.py compilemessages -l en",
```

Comment:
Good catch :+1:

:question: Though, same here: Shouldn't these two commands produce one `*.mo` file each, thus two in total? I see [only one](https://github.com/geometalab/osmaxx/pull/798/files#diff-562dd48f7010d93324f62151cc1b2126) being newly tracked. Are they overwriting each other's output?